code/poolawl.c

```diff
@@ -1057,7 +1057,7 @@ static void awlSegReclaim(Seg seg, Trace trace)
   Bool hasBuffer = SegBuffer(&buffer, seg);
   Format format = pool->format;
   Count reclaimedGrains = (Count)0;
-  Count preservedInPlaceCount = (Count)0;
+  STATISTIC_DECL(Count preservedInPlaceCount = (Count)0)
```

Comment:
q. Do these STATISTIC macros get compiled in (and therefore checked) for any of our varieties?

Reply:
Answer: yes, in COOL. https://github.com/Ravenbrook/mps/blob/4f5f119b23c92212f2647de383e13aee07bb2c1b/code/config.h#L50-L54